app/controllers/govuk_publishing_components/component_guide_controller.rb

```diff
@@ -78,6 +79,10 @@ def components_in_use_docs
       @components_in_use_docs ||= ComponentDocs.new(gem_components: true, limit_to: components_in_use)
     end
 
+    def components_not_in_use_docs
+      @components_not_in_use_docs ||= ComponentDocs.new(gem_components: true, limit_to: components_in_use)
+    end
+
```

Comment:
```suggestion
    def components_not_in_use_docs
      @components_not_in_use_docs ||= ComponentDocs.new(gem_components: true, limit_to: components_not_in_use)
    end
```

I think this should be `components_not_in_use` for this method